core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -140,7 +140,7 @@ var TopicsAndSkillsDashboardPage = function() {
     });
   };
 
-  this.createTopic = function(title, abbreviatedName) {
+  this.createTopic = function(title, abbreviatedName, closeTopicEditor) {
```

Comment:
Generally we use verbs for functions and nouns for variables. Maybe something like `shouldCloseTopicEditor`?

Reply:
Done